wagtail/admin/views/chooser.py

```diff
@@ -93,7 +93,7 @@ def can_choose_page(
 
             if user_perm == "move_to":
                 return page_to_move.permissions_for_user(user).can_move_to(page)
-    if user_perm == "copy_to":
+    if user_perm in {"add_subpage", "copy_to"}:
         return page.permissions_for_user(user).can_add_subpage()
```

Comment:
Unfortunately this doesn't take into account the allowed `subpage_types` and things like `max_count_per_parent`. Adding these checks here is quite complicated as it involves passing the params to the JS code, to later be used by the chooser views. I think for now we can validate this via the form instead.